### PR TITLE
Chain of Thrayn

### PR DESCRIPTION
**Pull request by CollCrom**
Added a chain of Thrayn Module

### CODE REVIEW

File: src/Parser/RetributionPaladin/Modules/Items/ChainOfThrayn.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+	damageDone = 0;
+
+	on_initialized() {
+		this.active = this.combatants.selected.hasWaist(ITEMS.CHAIN_OF_THRAYN.id);
+	}
+
+	on_byPlayer_damage(event) {
+		if(this.combatants.selected.hasBuff(SPELLS.CRUSADE_TALENT.id) || this.combatants.selected.hasBuff(SPELLS.CRUSADE_TALENT.id)){
+			this.damageDone += ((event.amount || 0) + (event.absorbed || 0)) * CHAIN_OF_THRAYN_INCREASE / (1 + CHAIN_OF_THRAYN_INCREASE);
```
Comment (anom0ly) [inline src/Parser/RetributionPaladin/Modules/Items/ChainOfThrayn.js]: You are checking for the `CRUSADE_TALENT.id` twice here, did you mean to select a different spell?

Also, isn't the 10% bonus from Thrayn already in the damage event? If so the formula would be `bonusDamage = raw - (raw / (1 + Increase))` I believe.

**Comment by anom0ly** [comment]
Just double check the logic on the damage increase contribution. Other than that looks good.

**Comment by CollCrom** [comment]
Yeah oops meant to do Avenging Wrath as well. I changed the calculation as well.

**Comment by anom0ly** [comment]
Looks good. Thanks.

**Pull request merged**